saleor/plugins/openid_connect/utils.py

```diff
@@ -412,7 +413,8 @@ def get_or_create_user_from_payload(
             defaults=defaults_create,
```

Comment:
Please make sure to set `is_confirmed=True` in defaults_create to be sure that User created with OIDC plugin don't need to confirm email.

I know it's set to true by deafult in field definition, but there;'s same story with `is_active` flag and this one is set explicitly.